bundles/automation/org.eclipse.smarthome.automation.module.core.test/src/test/groovy/org/eclipse/smarthome/automation/module/core/internal/RuntimeRuleTest.groovy

```diff
@@ -229,9 +228,13 @@ class RuntimeRuleTest extends OSGiTest{
         //TEST RULE
         waitForAssert({
             assertThat ruleRegistry.getStatus(rule.uid), is(RuleStatus.IDLE)
-        }) 
+        })
+
+        def EventPublisher eventPublisher = getService(EventPublisher)
+        eventPublisher.post(ItemEventFactory.createStateEvent("myMotionItem3", OnOffType.ON))
+        waitForAssert({
+            assertThat ruleRegistry.getStatus(rule.uid), is(RuleStatus.RUNNING)
```

Comment:
Wouldn't be more correct if there is a check if the event for rule state is received, rather than checking the temporary running state of the rule - this will avoid timing issues